### ISSUE TITLE
Corruptive Decay

### ISSUE THREAD

**Issue by pilotmans** [Studio-Plus/Evil-Mod-Public-Board#1]
**Feature description**
An Object with the resemblance of a blue glowing and sparking orb

**Feature purpose**
Any living entity which comes in a 3-meter radius of it without proper protection will be inflicted with "Corruption" and "Decay", this would cause the body to decay in an odd way such as limbs falling off whilst the guy is completely fine or just not bleeding but they will eventually rot away. Next step is where the body will revive itself in a corrupted form with high durability and strength.

**Additional context**
If a person physically touches it even with proper protection, they will either rot away or be teleported to another realm with slight distortion. This object would also be capable of producing power equivalent to "5 toggle buttons connected to a signal-to-electricity transformer." The protection against this hazardous orb would probably either be insulator or a whole new custom suit/material just for personal protection against this orb.

**Comment by Dawid23-Gamer** [comment]
Corruption in itself is already in the mod as a feature in multiple forms, the main ones being corrupted blood and corrupted/evil humans being able to bite others.
The mod and the world that it creates has its rules too so I'll think about it and how it could fit in.
TL;DR: I'll think about it
